pkg/config/linters_settings.go

```diff
@@ -368,6 +369,10 @@ func (cfg GovetSettings) Validate() error {
 	return nil
 }
 
+type GrouperSettings struct {
+	Config map[string]map[string]interface{} `mapstructure:"config"`
```

Comment:
`config` or `settings`? What's the preferred name?

Reply:
The configuration must be explicit:
```go
type GrouperSettings struct {
	ImportRequireSingleImport bool `mapstructure:"import-require-single-import"`
	ImportRequireGrouping     bool `mapstructure:"import-require-grouping"`
}
```